setup/FSharp.SDK/FSharp.SDK.wxs

```diff
@@ -2,7 +2,7 @@
 <!-- Copyright (c) Microsoft Corporation.  All Rights Reserved.  See License.txt in the project root for license information. -->
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi">
 
-  <?define ProductVersion = "4.1"?>
+  <?define ProductVersion = "4.3"?>
```

Comment:
Why don't we use "F# 4.1.3" here?  It is really, really confusing to be adding a new series of SDK version numbers which are so immensely confusing.